lib/airbrake/action_cable.rb

```diff
@@ -0,0 +1,20 @@
+module ActionCable
+  module Channel
+    class Base
+      alias perform_action_without_airbrake perform_action
+
+      def perform_action(*args, &block)
+        perform_action_without_airbrake(*args, &block)
+      rescue Exception => exception # rubocop:disable Lint/RescueException
+        Airbrake.notify(exception) do |notice|
+          notice.stash[:rack_request] = ActionDispatch::Request.new(connection.env)
```

Comment:
That's not a rack request. Is that a problem?

Reply:
Alternatively You I can put here `connection.send(:request)` but I guess it's not better that way.

Reply:
Might be a problem because our other filters read this data and expect this to be a request. What's connection's class?

Reply:
Same as I created, it's `ActionDispatch::Request`. Look here for reference: https://github.com/rails/rails/blob/master/actioncable/lib/action_cable/connection/base.rb#L152

Reply:
Since we already stash connection on the next line, wouldn't it make sense to leave only that. Why do we stash `connection.env` in the first place?

Reply:
I removed stashing rack_request